README.md

```diff
@@ -1,52 +1,167 @@
 
 ## Ajax
 
-describe what ajax is and what it's used for
+*** SOMETHING HERE to make it flow better ***
+
+Asynchrounous Javascript and XML (Ajax) is a technique that is used in web applications.  It provides a way for content to be retrieved from a server and displayed without having to refresh the entire page.  Remember in your **blank application** where you had to create a list of **objects**?  Each time we filled out the form and clicked on **create**, the web browser would have to refresh the entire page to see the results sent back from our server.
+
+Single page applications provide a better user experience by allowing users to manipulate data on the server and see the results without having to refresh the page.  This is the power of Ajax in action.  In the background, requests are made to a web API using JavaScript.  As developers we can then choose to alter the displayed HTML based on the responses from the web API.
+
+Let's try an example. To start, create a folder on your computer called ```example```. Inside that folder create another folder called ```html```. Now create a file named ```index.html```, ```script.js``` and ```html/sentence.html```. Finally add the following to each file.
+
+#### index.html
+```html
+<!doctype html>
+<head>
+  <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.3/jquery.min.js"></script>
+    <script src="script.js"></script>
+</head>
+<body>
+  <p id="sentences">
+    Loading our page...
+  </p>
+</body>
+```
+
+#### html/sentence.html
+```html
+<p>Ajax is a brand of cleaning products, introduced by Colgate-Palmolive in 1947 for a powdered household and industrial cleaner.</p>
+<p>It was one of the company's first major brands.</p>
+```
+
+#### javascript.js
+```javascript
+// We should wait for the page to load before running our ajax request
+$(document).ready(function(){
+  // Now we start the Ajax Get request. The first parameter is the URL with the data.
+  // The second parameter is a function that handles the response.
+  $.get("html/sentence.html", function(response) {
+    // Here we are getting the element on the page with the id of sentences and
+    // insert the response
+    $("#sentences").html(response);
+  });
+});
+```
+Using your terminal run the following command from inside the examples folder.
+
+```
+$ python -m SimpleHTTPServer
+```
+
+This starts a simple server that will serve our files over HTTP. Browse to [http://localhost:8000](http://localhost:8000/) and watch as the Ajax request is made and the new data is added to our web page. Pretty cool! We used the power of Ajax to load data from the ```html/sentence.html```. This same idea can be applied to calling the GitHub API or our Rails application.
```

Comment:
They will know events. So maybe have them hit a button that loads the code remotely? Because everything is local it'll be hard to point out that difference. It'll seem instant! Also show them that when you hit the button, in your terminal `SimpleHTTPServer` registers the call to `sentence.html`. Only then though